Expensify/App
# Update `Get started` button to `Finish setup` when a user has draft data saved in the VBA flow

### Details
The button in the Workspace card page should change to Finish setup when a user has entered data into the form and closes the modal.

### Fixed Issues
$ Expensify/App#4892

### Tests
1. Login to an account with a Workspace and without an Expensify Card enabled or bank account setup.
2. Navigate to `Settings > Select Workspace > Get Started`.
3. Start filling out the form.
4. Close the form and notice that the button changed to `Finish setup`.

### QA Steps
Steps above.

### Tested On

### Screenshots
#### Web

https://user-images.githubusercontent.com/22219519/135545264-d7c246cc-2509-4f4d-b4de-ea5a475aab83.mov

#### Mobile Web

#### Desktop

https://user-images.githubusercontent.com/22219519/135545274-d550ea37-0d8e-4575-9c22-d6fe846b043e.mov

#### iOS
#### Android
